modules/docs/sc_param.md

```diff
@@ -36,76 +36,76 @@ The sc_param module provides methods to help you handle parameters for your stre
+| send_data_test                          | number | 0                                                                             | When set to 1, send data in the logfile of the stream connector instead of sending it where the stream connector was designed to                                                                                                                                                                                                              | all                                                                                                                                                                                                                                                                                      |                                                                                                                                                                                                                                                                                                  |
+| format_file                             | string |                                                                               | Path to a file that will be used as a template to format events instead of using default format                                                                                                                                                                                                                                               | only usable for events stream connectors (\*-events-apiv2.lua) and not metrics stream connectors (\*-metrics-apiv2.lua) you should put the file in /etc/centreon-broker to keep your broker configuration in a single place. [**See documentation for more information**](templating.md) |                                                                                                                                                                                                                                                                                                  |
```

Comment:
```suggestion
| format_file                             | string |                                                                               | Path to a file that will be used as a template to format events instead of using the default format                                                                                                                                                                                                                                               | only usable for events stream connectors (\*-events-apiv2.lua) and not metrics stream connectors (\*-metrics-apiv2.lua) you should put the file in /etc/centreon-broker to keep your broker configuration in a single place. [**See documentation for more information**](templating.md) |                                                                                                                                                                                                                                                                                                  |
```